src/Storage/TSStorageManager.m

```diff
@@ -456,16 +448,15 @@ - (void)deleteDatabaseFile {
     }
 }
 
-- (void)wipeSignalStorage {
+- (void)resetSignalStorage {
     self.database = nil;
+    _dbConnection = nil;
```

Comment:
Sure, that sounds quite reasonable.  In fact, the only reason I accessed the ivar here is because this property is declared as `readonly` in the header and there's no `readwrite` property as one might expect in a private anonymous category.  Going forward I'll fix these as I find them!